apps/coscrad-frontend/src/components/AllResources/ResourceInfoPresenter/ResourceInfo.presenter.tsx

```diff
@@ -11,22 +20,26 @@ export const ResourceInfoPresenter = ({
  * TODO [https://www.pivotaltracker.com/story/show/183766033
  * We should expose the Schema as part of our own API docs somehow.
  */
-IAggregateInfo<ResourceType>): JSX.Element => {
-    const resourceIndexLink = (
-        <Link to={`/${routes.resources.ofType(resourceType).index}`}>View {label}s</Link>
-    );
-
-    return (
-        <>
-            <h3>
-                {label} ({resourceType})
-            </h3>
-
-            <div className="resource-meta" data-testid={label}>
-                {resourceIndexLink}
-
-                <div>{description}</div>
-            </div>
-        </>
-    );
-};
+IAggregateInfo<ResourceType>): JSX.Element => (
```

Comment:
I refactored to show the changes I'd had in mind on my end.
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/63014783/204694521-a9e000ad-9fc0-44fa-b57b-e431dd652229.png">